default/buildings.txt

```diff
@@ -1730,31 +1736,22 @@ BuildingType
                     WithinDistance distance = 100 condition = Source
                     System
                 ]
-                Destroy
```

Comment:
Really unsure about letting it stay in place after conquest and creating beasties for the player. You sometimes get a few on the turn it destroys itself and they're hideous if they're under sentient control.

Also, won't the whole 'not destroying' thing mean the Sitrep is triggered every turn? It's bad enough that Transcendance does that.